source/Logger.cpp

```diff
@@ -370,6 +370,7 @@ void FPSciLogger::logTargetTypes(const Array<shared_ptr<TargetConfig>>& targets)
 			"'" + config->destSpace + "'",
 			String(std::to_string(config->size[0])),
 			String(std::to_string(config->size[1])),
+			String(std::to_string(config->symmetricEcc)),
```

Comment:
I believe this adds the new parameter to each row added to the table, but I don't think the column labels have been updated, so this is likely to cause a problem. We'll need to make sure the column list gets updated before merging this PR.